articles/azure/azs-how-create-S2S-VPN-azurestack-to-azure.md

```diff
@@ -114,7 +116,7 @@ To associate a virtual network with a gateway, it must first contain a valid gat
 
    - **Name** - The name of the virtual network gateway.
 
-   - **SKU** - Route-based VPN gateway types are offered in three SKUs: Basic, Standard, and High performance. You must select Standard or High performance if you are creating the network to coexist with an ExpressRoute gateway. You must select High performance SKU to enable active-active mode. You can find more information about SKUs here: [Azure Stack Hub SKUs](https://docs.microsoft.com/en-gb/azure/vpn-gateway/vpn-gateway-about-skus-legacy).
+   - **SKU** - Route-based VPN gateway types are offered in three SKUs: Basic, Standard, and High performance. From version 1910 onwards, a custom IPSec policy is required for Azure Stack to connect to public Azure, therefore requiring a Standard or High SKU. If you are creating the network to coexist with an ExpressRoute gateway, you must select Standard or High also. You must select High performance SKU to enable active-active mode. You can find more information about SKUs here: [Azure Stack Hub SKUs](https://docs.microsoft.com/en-gb/azure/vpn-gateway/vpn-gateway-about-skus-legacy).
```

Comment:
We're not setting any custom IPSec policies on the Azure Stack Hub side, so we don't need this comment here.
```suggestion
   - **SKU** - Route-based VPN gateway types are offered in three SKUs: Basic, Standard, and High performance. If you are creating the network to coexist with an ExpressRoute gateway, you must select either Standard or High performance. If you want to enable active-active mode, you must only select High performance. You can find more information about SKUs here: [Azure Stack Hub SKUs](https://docs.microsoft.com/en-gb/azure/vpn-gateway/vpn-gateway-about-skus-legacy).
```

Reply:
Need to explain that the Stack uses legacy SKUs, which differs to public Azure